syntax = "proto3";

package data_stream;

import "google/protobuf/timestamp.proto";

service DataStream{

 rpc DataStreamingService(Data) returns (stream DataResponse) {}

}

message Data{
  int64 client_id = 1;
}

message DataResponse{
  int64 server_id = 1;
  string response_data = 2;
  google.protobuf.Timestamp timestamp = 3;
}
